sdk/lusid/api/derived_transaction_portfolios_api.py

```diff
@@ -91,7 +91,7 @@ def create_derived_portfolio_with_http_info(self, scope, **kwargs):  # noqa: E50
 
         local_var_params = locals()
 
-        all_params = ['scope', 'create_derived_transaction_portfolio_request']  # noqa: E501
+        all_params = ['scope''create_derived_transaction_portfolio_request']  # noqa: E501
```

Comment:
What happened to the comma here?

Reply:
Different variables names in the mustache template!